packages/react/context-menu/src/ContextMenu.stories.tsx

```diff
@@ -73,138 +59,160 @@ export const Modality = () => {
     <div
       style={{ display: 'flex', alignItems: 'center', justifyContent: 'center', height: '110vh' }}
     >
-      <div style={{ display: 'grid', gridGap: 50 }}>
+      <div style={{ display: 'grid', gap: 50 }}>
```

Comment:
`gripGap` was deprecated long ago and no longer renders with that syntax in latest Stitches